src/utils/useDocumentTitle.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+
+import playroomConfig from '../config';
+
+interface UseDocumentTitleProps {
+  title?: string;
+  suffix?: string;
+}
+
+export function useDocumentTitle({ title, suffix }: UseDocumentTitleProps) {
+  useEffect(() => {
+    document.title = getTitle({ title, suffix });
+  }, [title, suffix]);
+}
+
+function getTitle({
+  title,
+  suffix = 'Playroom',
+}: UseDocumentTitleProps): string {
+  if (title) {
+    return `${title} | ${suffix}`;
+  }
+
+  const configTitle = playroomConfig?.title;
+
+  if (configTitle) {
+    return `${configTitle} | ${suffix}`;
+  }
+
+  return suffix;
+}
```

Comment:
Sorry, feel like this function can be simplified to make it clearer what is happening. Thoughts?

```suggestion
function getTitle({
  title,
  suffix = 'Playroom',
}: UseDocumentTitleProps): string {
  const resolvedTitle = title || playroomConfig?.title;
  if (resolvedTitle) {
    return `${resolvedTitle} | ${suffix}`;
  }
  
  return suffix;
}
```

Reply:
Yep can be, thought that's a bit harder to parse all the cases but happy to change